lib/widgets/package_tile.dart

```diff
@@ -15,70 +18,51 @@ class PackageTile extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return GroovinExpansionTile(
+    Color scoreColor;
+    if (page.packages[index].score <= 50 || page.packages[index].score == null) {
+      scoreColor = Provider.of<PubColors>(context).badPackageScore;
+    } else if (page.packages[index].score >= 51 && page.packages[index].score <= 69) {
+      scoreColor = Provider.of<PubColors>(context).goodPackageScore;
+    } else {
+      scoreColor = Provider.of<PubColors>(context).greatPackageScore;
+    }
+    return ListTile(
       title: Text(
         page.packages[index].name,
         style: TextStyle(
           fontWeight: FontWeight.bold,
         ),
       ),
       subtitle: Text(
-        'v${page.packages[index].latest.version.major}'
-        '.${page.packages[index].latest.version.minor}'
-        '.${page.packages[index].latest.version.patch}'
-        ' updated ${page.packages[index].dateUpdated}',
+        'v${page.packages[index].latest.semanticVersion.major}'
```

Comment:
Let's refactor this into a variable 
`final package = page.packages[index].latest;`

Reply:
yeah was gonna get there eventually :)